src/components/Button.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styled from "styled-components";
+
+const Button = ({ onClick, children, type = "button" }) => {
+  return (
+    <MainButton onClick={onClick} type={type}>
+      {children}
+    </MainButton>
+  );
+};
+
+const MainButton = styled.button`
+  color: ${(props) => (props.secondary ? "gray" : "white")};
```

Comment:
You could also deconstruct the object, same in the next lines. This is ok though :)

Reply:
Like said in the other PR; I don't see the `<MainButton>` component taking any prop named `secondary`.
Also; best to store the colors in a theme object and expose it through the app with the `<ThemeProvider>`